52.md

```diff
@@ -49,6 +46,8 @@ The following tags are deprecated:
     ["d", "<UUID>"],
 
     ["title", "<title of calendar event>"],
+    ["image", "<image url>"],
```

Comment:
Oh, these look new. I guess @zmeyer44 added them to Flockstr. We should add documentation in the date-based and time-based calendar events on what the `image` and `banner` tags are for both kinds.